docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
       context: elasticsearch/
     volumes:
       - ./elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro
+      - ./elasticsearch/config/sg:/usr/share/elasticsearch/config/sg:ro
```

Comment:
This is not enough, you should also remove all related statements from the Dockerfile.